modules/core/src/main/scala/org/scalasteward/core/update/PruningAlg.scala

```diff
@@ -46,9 +46,14 @@ final class PruningAlg[F[_]](
     repoCacheRepository.findCache(repo).flatMap {
       case None => F.pure((false, List.empty))
       case Some(repoCache) =>
+        val ignoreScalaDependency =
+          !repoCache.maybeRepoConfig.flatMap(_.updates.includeScala).getOrElse(false)
```

Comment:
Could you add `UpdatesConfig.defaultIncludeScala: Boolean` and use that instead of `false` here? This way all default config values are defined under the `repoconfig` package which makes it easier to figure out what the default values of the various config options are.

Reply:
updated, thanks! let me know if the change is as intended

Reply:
Sorry, I should have been more clear. What I had in mind was this:
```scala
case class UpdatesConfig(
  ...
  includeScala: Option[Boolean]
)

object UpdatesConfig {
  ...
  val defaultIncludeScala: Boolean = false
}

// and then in PruningAlg
!repoCache.maybeRepoConfig.flatMap(_.updates.includeScala)
  .getOrElse(UpdatesConfig.defaultIncludeScala)
```

The optional `includeScala` field in the `UpdatesConfig` class was fine. If this is changed to a `Boolean` with a default value, it means that the default value will be persisted in `RepoCache` objects by `RepoCacheAlg`. But persisted default values make it harder to change the default later because we can't differentiate between a persisted default value and the same value that was explicitly set by the user. (#1339 and #1344 has been done for the same reason.)

Reply:
fixed, thanks for the detailed explanation